src/pages/WatchlistPage.js

```diff
@@ -8,14 +8,14 @@ export default function WatchlistPage({ watchlist }) {
   return (
     <Wrapper>
       <Header />
-      {watchlist.length === 0 ? (
+      {watchlist?.length === 0 ? (
         <InfoBox>
           <p>👻</p>
           <p>Deine Watchlist ist aktuell leer.</p>
         </InfoBox>
       ) : (
         <StyledList role="list">
-          {watchlist.map(({ name, title, posterPath, id }) => (
+          {watchlist?.map(({ name, title, posterPath, id }) => (
```

Comment:
You don't need the optional chaining here. You already check for the length of the watchlist in your ternary statement.

Reply:
Oh... yes you're right! Thanks! 🙂

Reply:
Little Update:
I need it, because the length check is for conditional rendering. Otherwise, my App crashes

Reply:
Hm, but only when the `watchlist` has a length the `map` will render. It seems a bit weird, that you then need to optional chaining again 🤔

Reply:
I'm a little surprised about that, too.
But if I leave optional chaning out, the console tells me `.map is not a function` because the data is not there yet.
Anyway, that's how it works now without crashing 🙂

Reply:
Okay 😅